.travis.yml

```diff
@@ -2,31 +2,85 @@ language: python
 sudo: false
 
 python:
-  - "2.7"
+- 2.7
 
 env:
-  - TOXENV=flake8
-  - TOXENV=docs
-  - TOXENV=assets
-  - TOXENV=es
-  - TOXENV=addons
-  - TOXENV=devhub
-  - TOXENV=editors
-  - TOXENV=amo
-  - TOXENV=users
-  - TOXENV=main
```

Comment:
Looking at https://docs.travis-ci.com/user/environment-variables/#Global-Variables you can re-use how we defined the environments earlier and don't need to re-define the whole matrix. Mind giving this a try?